js/ssai_pulse.js

```diff
@@ -43,6 +43,7 @@ OO.Ads.manager(function(_, $)
     this.currentAd = null;
 
     var amc  = null;
+    var currentOffset = 0;
```

Comment:
Do we need to reinitialize currentOffset anywhere (like on replays, content change, etc)?